src/helm/benchmark/static/schema.yaml

```diff
@@ -404,6 +404,14 @@ models:
     num_parameters: 6700000000
     release_date: 2022-11-29
     todo: true
+  - name: together/gpt-neoxt-chat-base-20b
+    display_name: GPT-NeoXT-Chat-Base (20B)
+    description: GPT-NeoXT-Chat-Base (20B) is a fork of GPT-NeoX.
```

Comment:
Can we be a bit more descriptive - it's fine-tuned to..

Reply:
updated.